insights/core/reducer.py

```diff
@@ -53,12 +53,12 @@ def collect_results_multi(results_dict):
     return combined_local_output, combined_shared_output
 
 
-def run_multi(parser_output, all_output, error_handler, reducers=plugins.CLUSTER_REDUCERS, stats=None):
+def run_multi(parser_output, all_output, error_handler, reducers=plugins.CLUSTER_REDUCERS, reducer_stats=None, ):
     logger.debug("Multi-node reducer run started")
     if reducers != plugins.CLUSTER_REDUCERS:
-        log_components(reducers)
+        log_reducers(reducers)
```

Comment:
Changing the name to include "reducers" seems to be going in the wrong naming direction.  Perhaps we should change the name on 1.x to `log_components`?